docker-compose/is/dockerfiles/is/Dockerfile

```diff
@@ -17,11 +17,11 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to WSO2 Identity Server Docker image with latest WSO2 Updates
-FROM docker.wso2.com/wso2is:5.11.0.0
+FROM docker.wso2.com/wso2is:6.0.0.0
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>"
 
 # build arguments for external artifacts
-ARG MYSQL_CONNECTOR_VERSION=8.0.29
```

Comment:
Why are we downgrading MYSQL connector version. AFAIK, 8.0.20 is vulnerable version